Sources/Core/Transaction/CodableTransaction.swift

```diff
@@ -11,7 +11,6 @@ import BigInt
 ///  while all fields in this struct are optional, they are not necessarily
 ///  optional for the type of transaction they apply to.
```

Comment:
Luckily not all, but most of course.

Reply:
I guess is the confusing one, I, be meant here that there's a typo within comment

> while **all** fields optional… 

While it is 

> while **most** fields are optional…